blog/test_app.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+from webtest import TestApp
+from google.appengine.ext import testbed
+
+from bz_blog.wsgi import application
+
+
+class HomepageTest(unittest.TestCase):
+    def setUp(self):
+        self.testbed = testbed.Testbed()
```

Comment:
I am following this tutorial, https://cloud.google.com/appengine/docs/python/tools/localunittesting